energy_analyse2020.10.16_213101.R

```diff
@@ -0,0 +1,177 @@
+#set path of data and get all the subforders
+#change the path where you put the data
+setwd("D:/Rdata/2020.10.16_213101/data/nexus6p") 
+#change the path where you put the data
+projectpath="D:/Rdata/2020.10.16_213101/data/nexus6p"
+
+library(stringr)
+
+#for app with critical
+criticalfiles<-list.files(pattern = "subjects_critical")
+name_cri_chro <- strsplit(criticalfiles, "-")
+name_cri_chro <- sapply(name_cri_chro, "[", 2)
+appbrowser_cri_chro <- paste(criticalfiles, "chrome", sep = "/")
+androidappbrowser_cri_chro<- paste(appbrowser_cri_chro, "android", sep = "/")
+trepnappbrowser_cri_chro<- paste(appbrowser_cri_chro, "trepn", sep = "/")
+batterystatsappbrowser_cri_chro<- paste(appbrowser_cri_chro, "batterystats", sep = "/")

```

Comment:
I don't understand what is going on here from reading the code and variable names. Could you add some comments to explain?

Reply:
Yes, I will update it in next version. For the  code here, it is just to make the common path into seperate folders like trepn or andriod folder